doc/sphinx/source/framework-integration-guides.rst

```diff
@@ -4,9 +4,61 @@
 Framework Integration Guides
 #############################
 
+* :ref:`neon_intg`
 * :ref:`mxnet_intg`
 * :ref:`tensorflow_intg`
 
+.. _neon_intg:
+
+Neon |trade|
+============
+
+Use ``neon`` as a frontend
+---------------------------
+
+``neon`` is a open source Deep Learning framework specifically designed to be 
+powered by |InG| backends.
+
+.. important:: The numbered instructions below pick up from where 
+   the :doc:`install` installation instructions left off, and they presume 
+   that your system already has the library installed at ``$HOME/ngraph_dist`` as 
+   the default location. If the |nGl| code has not yet been installed to your 
+   system, you can follow the instructions on the `ngraph-neon python README`_ to 
+   install everything at once.  If the |nGl| code already is installed, 
+
+
+
+#. Set the ``NGRAPH_CPP_BUILD_PATH`` and the ``LD_LIBRARY_PATH`` path to the location 
+   where you built the nGraph libraries:
+
+   .. code-block:: bash
+
+      export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist/
+      export LD_LIBRARY_PATH=HOME/ngraph_dist/lib/       
```

Comment:
Missing $ for parameter HOME in export LD_LIBRARY_PATH=HOME/ngraph_dist/lib/

Reply:
Good catch, thanks.  :+1: